jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ImageToJsonTranslator.java

```diff
@@ -67,6 +69,27 @@
     return result.build();
   }
 
+  /**
+   * Converts the map of environment variables to a list with items in the format "NAME=VALUE".
+   *
+   * @return the list
+   */
+  @VisibleForTesting
+  @Nullable
+  static ImmutableList<String> environmentMapToList(@Nullable Map<String, String> environment) {
+    if (environment == null) {
+      return null;
+    }
+    Preconditions.checkArgument(
+        environment.keySet().stream().noneMatch(key -> key.contains("=")),
```

Comment:
Or, even more clear may be s/key/variableName/

Reply:
I like `key` more since it's descriptive about which part of the map itself it's dealing with. And we can still tell that the key corresponds to the name with the error message right below.